2996_reflection/reflection.md

```diff
@@ -5556,6 +5557,44 @@ int main() {
 :::
 :::
 
+### [temp.dep.temp]{.sref} Dependent template arguments {-}
+
+Add a new paragraph to cover dependent splice template arguments.
+
+::: std
+[4]{.pnum} A template `$template-parameter$` is dependent if it names a `$template-parameter$` or if its terminal name is dependent.
+
+[[5]{.pnum} A splice template argument is dependent if its `$splice-specifier$` is dependent.]{.addu}
+
+:::
+
+### 13.8.3.6 [temp.dep.alias] Dependent aliases {-}
+
+Add a new section to cover dependent type aliases and namespace aliases.
+
+::: std
+::: addu
+[1]{.pnum} A type alias is dependent if its underlying entity is a dependent type.
+
+[2]{.pnum} A namespace alias is dependent if it is introduced by a `$namespace-alias-definition$` containing a dependent `$nested-name-specifier$` or a dependent `$splice-specifier$`. A `$namespace-name$` is dependent if it names a dependent namespace alias.
+
+::: example
+```cpp
+template <std::meta::info R> int fn() {
+  namespace Alias = [:R:];  // [:R:] is dependent.
+  return Alias::v;
```

Comment:
Should we also note in the comment that `Alias` is dependent too, since that's what the above is saying.

Reply:
Yep, done.